src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package mate.jdbc.service.impl;
+
+import java.sql.SQLException;
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id)
+                .orElseThrow(() -> new DataProcessingException("Can't get driver by id = ["
+                        + id + "]", new SQLException()));

```

Comment:
Just return `NoSuchElementException` with the message here. Why you need to create empty `SQLException`?